app/views/dit_landing_page/_page_header.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :is_full_width_header, true %>
+<% content_for :title, t("dit_landing_page.meta_title") %>
+<% content_for :meta_tags do %>
+  <meta property="og:type" content="website" />
+  <meta property="og:site_name" content="GOV.UK">
+  <meta property="og:url" content="<%= request.base_url + request.path %>">
+  <meta property="og:title" content="<%= t("dit_landing_page.meta_title") %>">
+  <meta property="og:description" content="<%= t("dit_landing_page.meta_description") %>">
+  <meta property="og:image" content="<%= image_url("KBM-logo-placeholder.png") %>" />
+  <meta name="twitter:card" content="summary">
+  <meta name="twitter:image" content="<%= image_url("KBM-logo-placeholder.png") %>">
+  <meta name="description" content="<%= t("dit_landing_page.meta_description") %>">
+  <meta name="govuk:navigation-page-type" content="Taxon Page" />
+<% end %>
+
+<header class="dit-landing-page__header--with-bg-image">
+  <div class="dit-landing-page__header-text-block">
+    <div class="govuk-width-container landing-page_header-text-container">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third">
+          <%= render "govuk_publishing_components/components/translation-nav", {
+            translations: @translation_links,
+            inverse: true
+          } %>
+        </div>
+      </div>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="dit-landing-page__page_title">
+            <%= t("dit_landing_page.page_header") %>
+          </h1>
+        </div>
+        <div class="govuk-grid-column-two-thirds dit-landing-page__intro">
+          <%= render "govuk_publishing_components/components/govspeak", {
```

Comment:
We do not need to pass this through govspeak given the plain text content. Unless we expect it to have links (which would require additional styling anyway) I would recommend a plain `<p class="govuk-body-l"><%= t("dit_landing_page.page_header_explainer") %></p>`.